lang/shared/src/main/scala/com/wavesplatform/lang/v1/evaluator/ctx/impl/waves/Bindings.scala

```diff
@@ -287,6 +291,23 @@ object Bindings {
         )
     }
 
+  def blockHeaderObject(header: BlockHeader): CaseObj =
+    CaseObj(
+      blockHeader,
+      Map[String, EVALUATED](
+        "timestamp"           -> header.timestamp,
+        "version"             -> header.version,
+        "reference"           -> header.reference,
+        "generator"           -> header.generator,
+        "generatorPublicKey"  -> header.generatorPublicKey,
+        "signature"           -> header.signature,
+        "baseTarget"          -> header.baseTarget,
+        "generationSignature" -> header.generationSignature,
+        "transactionCount"    -> header.transactionCount,
```

Comment:
Is transactionCount field really needed?

Reply:
We can remove it later (now it's not clear do we need this filed or not)